File: CLab/PT_FACT.c
#include"stdio.h"
int Fact(int);
void main()
{
	int num;
	printf("enter max range:-");
	scanf("%d",&num);
	int res=Fact(num);
	printf("%d",res);

}
int Fact(int n)
{
	if(n>=1)
		return (n*Fact(n-1));
	else
		return 1;
}

File: PythonLab/PT_Fact.py
"""
def fun(num):
	if(num>=1):
		return(num*fun(num-1))
	else:
		return(1)
num=int(input("enter max limits"))
res=fun(num)
print(res)
"""
fact=1
num=int(input("enter max limits"))
for x in range(1,num+1):
	fact=fact*x
print(fact)
